### PR TITLE
VACMS-11435: removed old table audit

### LINKED ISSUE #11435: Table audits cleanup
## Description
There are a few issues with the Table audits.  
that should be cleaned up for consistency.

## Acceptance Criteria
- [ ] Tabs should be alpha sorted 
image
- [ ] we don't have  (New) pattern on tabs anywhere.  That will become outdated fast. Can we remove it?
- [ ] The original Tables audit seems to not speak the truth about the number of tables.  Its only value seems to be that it has filters where the new one doesn't.  Can we implement the filters on the new one, and remove the old completely?
- [ ] One of the Views style guide criteria is that we use meaningful machine names on our View displays instead of the generic defaults.  Can we update the machine names to mimic the original View machine names?
image
- [ ] The View needs a Caption that tells us what we are looking at.
image

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by edmund-dunn**
## Description

Closes #11435.

## Testing done
Tested locally.

## Screenshots
CleanShot 2022-11-17 at 11 22 54

## QA steps

Navigate to `/admin/content/audit/tables`. The old view is gone. The new Table audit is now labeled "Tables", `(New)` has been removed. The audit tabs are now in alphabetical order. 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/views.view.tables.yml [hunk 1]
```diff
@@ -641,7 +641,7 @@ display:
           admin_label: ''
           plugin_id: result
           empty: false
-          content: 'Displaying @start - @end of @total'
+          content: "<p>Displaying @start - @end of @total</p>\r\n<p>This audit shows all of the tables used on the site. </p>"
```
Comment (swirtSJW) [inline config/sync/views.view.tables.yml:644]: Should this added text be in the View caption instead?

Comment (edmund-dunn) [reply config/sync/views.view.tables.yml:644]: None of the audit views except the Buttons audit had anything like a caption. This is how it was done there. I was just following the pattern I could find.

Comment (laflannery) [reply config/sync/views.view.tables.yml:644]: The table caption is something that is missing from the majority of our tables so as we touch these views/tables we have been trying to add these captions in.  Christian added captions to the [facilities tables here](https://pr11638-da3uxe6mpxoicgevb2mu7o3b3yhk7hpx.ci.cms.va.gov/admin/content/facilities)

Can you remove the text that says "This audit shows all of the tables used on the site" and instead add a caption for "List of all tables used on the site"?

Comment (edmund-dunn) [reply config/sync/views.view.tables.yml]: @laflannery Ok, that makes sense. Thank you and done!

**Review by laflannery — APPROVED** [review]
This looks good to me, just make sure @swirtSJW approves as well because that was his comment/request

**Pull request merged**